clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2546,6 +2555,11 @@ generated code appropriately for the kernel to which attribute is applied.
 While semantic of this attribute is the same between OpenCL and SYCL,
 spelling is a bit different: 
 
+When conforming to SYCL 1.2.1 specifications, when called from a device kernel,
+the ``cl::reqd_work_group_size`` is propagated from the function it is
```

Comment:
```suggestion
the ``sycl::reqd_work_group_size`` is propagated from the function it is
```
ah, this one is in the spec :-)

Reply:
We should be careful here -- that spelling doesn't exist until https://github.com/intel/llvm/pull/3689 lands, and we need to document the behavior of `cl`, `intel`, and `sycl` vendor namespaces explicitly as part of this patch.